### DISCUSSION TITLE
Cerbo GX  cp: omitting directory

### DISCUSSION THREAD

**Discussion by Mindi65** [Louisvdw/dbus-serialbattery#227] [Q&A] [answered]
Hi
<img width="427" alt="Screenshot 2022-09-30 210102" src="https://user-images.githubusercontent.com/114769077/193330219-d637d6ac-d6ab-4c0b-be0c-7851afd7d14b.png">

Few weeks ago I installed software on Cerbo GX, but had no RS485 to USB converter. When tied to test with it today any command I run I get 
cp: omitting directory '/data/etc/dbus-serialbattery/service'

I ast as superuser

**Comment by Louisvdw** [comment, marked as answer]
You can ignore that service subfolder. It only copies the very first time and then gets ignored

Once installed, reboot your GX and then look at the log files to see what is happening if it does not connect by itself.
https://github.com/Louisvdw/dbus-serialbattery/wiki/Troubleshoot

**Reply by Mindi65** [reply]
root@einstein:~# tail -f /data/log/dbus-serialbattery.ttyUSB0/current
tail: can't open '/data/log/dbus-serialbattery.ttyUSB0/current': No such file or directory
tail: no files

<img width="526" alt="Screenshot 2022-09-30 225803" src="https://user-images.githubusercontent.com/114769077/193347161-ff58582f-edae-45d4-bc88-e30b5093a2b5.png">

**Reply by Louisvdw** [reply]
What VenusOS firmware version do you have?
Try the new v0.13 that was released yesterday.

**Reply by Mindi65** [reply]
Yee it works!

**Reply by Mindi65** [reply]
After FW upgrade how to make it work again? 
Any process to start?

**Reply by Mindi65** [reply]
It worked before update.

**Reply by Louisvdw** [reply]
just reboot and it should install itself again after firmware upgrades

**Reply by Mindi65** [reply]
rebooted and it works again, THANKS

If there will be new driver version, what is procedure to update it?

**Reply by Louisvdw** [reply]
Exacly what you did today. You just run the installer and reboot

**Comment by Mindi65** [comment]
Thanks for your great work!
